app/services/editions/IssueQueries.scala

```diff
@@ -193,13 +194,15 @@ trait IssueQueries {
           val collectionsForFront = rows
             .flatMap(row => row.collection.filter(_.frontId == front.id))
             .sortBy(_.index)
+            .map(_.collection)
+            .foldLeft(List.empty[EditionsCollection]) {(acc, cur) => if(acc.exists(f => f.id == cur.id)) acc else acc :+ cur}
```

Comment:
Is this a case for the `distinctBy` that you used before or am I missing something?

Reply:
`distinctBy` was added in scala 2.13 :(

Reply:
In another PR i've written an implicit class that enriches `distinctBy` onto `Collection` — I'll see if our libraries are compiles against 2.13 but I'm not going to hold my breath lmao

Reply:
No problems, was just double checking as I was just trying to work out what it did. No bigggggy.